_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -111,6 +113,22 @@ WAI-ARIA states and properties with empty value, specified as an empty string, a
 <div role="combobox" aria-controls="">My combobox</div>
 ```
 
+#### Passed Example 9
+
+This `aside` element has an [explicit role][] of `none`. However, the [global][] [property][] `aria-label` is specified. Thus it has a [semantic role][] of `complementary` due to [Presentational Roles Conflict Resolution][]. The `aria-expanded` [state][] is [inherited][] for the `complementary` role.
+
+```html
+<aside role="none" aria-label="About ACT rules" aria-expanded="true">ACT rules are cool!</aside>
```

Comment:
The behavior of AT here is not consistent. JAWS does not report the expanded state. NVDA only reports it in Firefox.

Reply:
@daniel-montalvo my problem when writing these examples was to find something:
- with an implicit role (to trigger conflict resolution vs `role="none"`); and
- the implicit role has some specific (non-global) ARIA attributes (because the global ones are always allowed and this rule is pointless for them); and
- the element is not focusable by default (because focusable takes precedence over the presentational role conflict resolution, so if the element is focusable it will be exposed anyway and that doesn't test the other part of the conflict resolution (exposing global attribute)) (this exclude things like `button` or links…); and
- the ARIA attribute is not set to a correct value by the AAM (because if it's already set, having it explicit is a bad practice and overwriting it to a non-default value is likely a bad practice also if there is no context to explain it) (this exclude things such as `aria-level` on headings, or `aria-posinset` on list which have correct default value).

With all these constraints, I had a somewhat limited choice… Another possibility would have been `<section><h1 aria-level=2>Foo</h1></section>` but it's in itself a bit tricky why this `h1` needs its level to be changed (and requires the `section` as context), so I ended up choosing this as a decent example where the code shown is simple and not bad practice…

I'll be happy to change it if you figure out some other case that would fit within the constraints above…